packages/office-ui-fabric-react/src/components/ColorPicker/ColorPicker.deprecated.test.tsx

```diff
@@ -0,0 +1,75 @@
+import * as React from 'react';
+import { mount, ReactWrapper } from 'enzyme';
+import * as ReactTestUtils from 'react-dom/test-utils';
+import * as WarnUtil from '@uifabric/utilities/lib-commonjs/warn';
+
+import { ColorPicker } from './ColorPicker';
+import { ColorPickerBase, IColorPickerState } from './ColorPicker.base';
+import { IColorPicker, IColorPickerProps } from './ColorPicker.types';
+import { IColor } from '../../utilities/color/colors';
+import { mockEvent } from '../../common/testUtilities';
+
+describe('ColorPicker deprecated', () => {
+  let wrapper: ReactWrapper<IColorPickerProps, IColorPickerState, ColorPickerBase> | undefined;
+  const colorPickerRef = React.createRef<IColorPicker>();
+
+  beforeAll(() => {
+    // Prevent warn deprecations from failing test
+    jest.spyOn(WarnUtil, 'warnDeprecations').mockImplementation(() => {
+      /** no impl **/
+    });
+  });
+
+  afterAll(() => {
+    jest.restoreAllMocks();
+  });
+
+  afterEach(() => {
+    if (wrapper) {
+      wrapper.unmount();
+      wrapper = undefined;
+    }
+  });
+
+  it('respects color prop change', () => {
+    const onChange = jest.fn();
+    wrapper = mount(<ColorPicker color="#abcdef" onColorChanged={onChange} componentRef={colorPickerRef} />);
+
+    wrapper.setProps({ color: '#AEAEAE' });
+    expect(colorPickerRef.current!.color.hex).toEqual('aeaeae');
+    // weird but preserves existing behavior
```

Comment:
Can we be more specific about what is "weird" about this behavior?

Reply:
Calling onChange when props are updated is weird. I'll add a comment.